network-api/networkapi/donate/factory/ways_to_give_page.py

```diff
@@ -0,0 +1,28 @@
+from factory import Faker
+from wagtail_factories import PageFactory
+
+from networkapi.donate.models import DonateLandingPage
+from networkapi.utility.faker import StreamfieldProvider
+from networkapi.utility.faker.helpers import reseed
+from networkapi.wagtailpages.models import OpportunityPage
+
+Faker.add_provider(StreamfieldProvider)
+
+streamfield_fields = ["paragraph", "linkbutton", "spacer", "quote"]
+
+
+class DonateWaysToGivePageFactory(PageFactory):
+    class Meta:
+        # Using OppurtunityPage AKA "Default Page" model
```

Comment:
Leaving this comment as there is no `WaysToGivePage` model. Instead, https://foundation.mozilla.org/en/donate/ways-to-give/ is a `DefaultPage`.